LuaRules/Gadgets/customCommands.lua

```diff
@@ -0,0 +1,55 @@
+local moduleInfo = {
+	name = "customCommandsGadget",
+	desc = "Gadget connecting custom commands module allowing players to define their own commands",
+	author = "PepeAmpere",
+	date = "2017-01-24",
+	license = "notAlicense",
```

Comment:
If it's notAlicense, you need to add a License file. But I think to avoid problem inside the group, you could change it to GPLv2.

Reply:
solved in [43c5d5f]